src/lib/components/drip-list-badge/drip-list-badge.svelte

```diff
@@ -3,53 +3,52 @@
   import ensStore from '$lib/stores/ens';
   import formatAddress from '$lib/utils/format-address';
 
-  export let listName: string;
   export let listId: string;
-  export let owner: string;
+  export let listName: string | '##loading##' | undefined = undefined;
```

Comment:
not a _huge_ fan of using a magic literal value with a user-defined string, but whatever, quite unlikely anyone will call their list this^^

Reply:
oki, added a listLoading flag instead